inference-engine/src/transformations/src/transformations/op_conversions/convert_sequences_to_tensor_iterator.cpp

```diff
@@ -234,13 +333,13 @@ ngraph::pass::ConvertRNNSequenceToTensorIterator::ConvertRNNSequenceToTensorIter
 
 ngraph::pass::ConvertGRUSequenceToTensorIterator::ConvertGRUSequenceToTensorIterator() {
     MATCHER_SCOPE(ConvertGRUSequenceToTensorIterator);
-    // X, H, seq_lengths - static, W,R,B - any
-    auto rnn_seq = ngraph::pattern::wrap_type<opset5::GRUSequence>({pattern::any_input(pattern::has_static_shape()),
-                                                                    pattern::any_input(pattern::has_static_shape()),
-                                                                    pattern::any_input(pattern::has_static_shape()),
-                                                                    pattern::any_input(),
-                                                                    pattern::any_input(),
-                                                                    pattern::any_input()});
+    // GRU Cell support only static ranks
+    auto gru_cell = ngraph::pattern::wrap_type<opset5::GRUSequence>({pattern::any_input(pattern::has_static_rank()),
+                                                                    pattern::any_input(pattern::has_static_rank()),
+                                                                    pattern::any_input(pattern::has_static_rank()),
+                                                                    pattern::any_input(pattern::has_static_rank()),
```

Comment:
why do we need static rank for W,R,B inputs?

Reply:
At the moment, nGraph sequence node will fall at the validation stage if you try to create it with a dynamic rank for inputs. But this behavior will be changed, so I agree: it is necessary to remove these requirements in the matcher

Reply:
Corrected, thanks!